src/Microsoft.DotNet.Build.Tasks/PackageFiles/Roslyn.Common.props

```diff
@@ -4,8 +4,11 @@
     <RoslynVersion>2.0.0-rc</RoslynVersion>
     <RoslynPackageName>Microsoft.Net.Compilers</RoslynPackageName>
     <RoslynTargetsPath>$(ToolRuntimePath)</RoslynTargetsPath>
-    <CSharpCoreTargetsPath Condition="'$(CSharpCoreTargetsPath)' == ''">$(RoslynTargetsPath)$(RoslynPackageName)\$(RoslynVersion)\tools\Microsoft.CSharp.Core.targets</CSharpCoreTargetsPath>
-    <VisualBasicCoreTargetsPath Condition="'$(VisualBasicCoreTargetsPath)' == ''">$(RoslynTargetsPath)$(RoslynPackageName)\$(RoslynVersion)\tools\Microsoft.VisualBasic.Core.targets</VisualBasicCoreTargetsPath>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
```

Comment:
@weshaggard Nope. I'm not sure we ever want to do that. I'll be porting the compiler server to work on .NET Core, at which point the performance difference should be eliminated. At that point, I think it would make sense for most people to be using the .NET Core compiler everywhere.

Bundling the desktop facades seems a waste of space and bandwidth for very little gain. Certainly it will still be available if you need specific functionality we haven't ported, but even the capability difference will decrease over time.